students/yechan/westagram/users/views.py

```diff
@@ -27,7 +28,7 @@ def post(self, request):
             User.objects.create(
                 username    = username,
                 email       = email,
-                password    = password,
+                password    = bcrypt.hashpw(password.encode('utf-8'), bcrypt.gensalt()),
```

Comment:
```suggestion
                password    = bcrypt.hashpw(password.encode('utf-8'), bcrypt.gensalt()).decode('utf-8'),
```
hashpw의 결과가 ecode된 값이 나옵니다. 따라서 DB에 저장하기 전에 다시 decode 되어야 합니다.